example/mixin/alerts.yaml

```
@@ -77,3 +77,14 @@ groups:
     for: 10m
     labels:
       severity: warning
+  - alert: ConfigReloaderSidecarApplyErrors
+    annotations:
+      description: |-
+        Errors encountered while the {{$labels.pod}} config-reloader sidecar attempts to apply config in {{$labels.namespace}} namespace.
+        As a result, configuration for service running in {{$labels.pod}} may be stale and cannot be updated anymore.
+      summary: config-reloader sidecar was unable to apply new config for 5m
```

```suggestion
      summary: config-reloader sidecar was unable to apply new config
```

Just nitpicking, but if someone overrides `FOR` statement in alert, this summary won't be correct anymore.